scripts/deploy/deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# This script runs the aws cli command to deploy the app to s3
+# It also uploads source maps to bugsnag
+
+# Try this step and throw an error if it fails
```

Comment:
Does `aws` throw an error if the operation fails? I noticed the script doesn't implement that.